src/components/MeetupList.tsx

```diff
@@ -110,29 +50,78 @@ export interface IProps {}
 const MeetupList: React.FC<IProps> = () => {
   const [searchField, setSearchField] = useState<string>("")
   const filteredMeetups = filterMeetups(searchField)
+  const { colorMode } = useColorMode()
 
   const handleSearch = (event: React.ChangeEvent<HTMLInputElement>): void =>
     setSearchField(event.target.value)
 
   return (
-    <div>
-      <StyledInput
+    <Box>
+      <Input
         onChange={handleSearch}
         placeholder={"Search by meetup title or location"}
+        display="block"
+        mr="auto"
+        ml="auto"
+        mb={6}
+        border="1px solid"
+        borderColor="searchBorder"
+        color="text"
+        bg="searchBackground"
+        p={2}
+        borderRadius="base"
+        w="100%"
+        _focus={{ outline: "solid 1px", outlineColor: "primary" }}
+        _placeholder={{ color: "text200" }}
       />
-      <Table>
+      <Box
+        boxShadow={colorMode === "dark" ? "tableBox.dark" : "tableBox.light"}
+      >
         {filteredMeetups.length ? (
           filteredMeetups.map((meetup, idx) => (
-            <Item key={idx} to={meetup.link}>
-              <LeftContainer>
-                <ItemNumber>{idx + 1}</ItemNumber>
-                <ItemTitle>{meetup.title}</ItemTitle>
-              </LeftContainer>
-              <RightContainer>
-                <Emoji text={meetup.emoji} size={1} mr={`0.5em`} />
-                <ItemDesc>{meetup.location}</ItemDesc>
-              </RightContainer>
-            </Item>
+            <LinkBox key={idx}>
+              <LinkOverlay
```

Comment:
So, the idea of this `LinkOverlay` is to wrap only the text you want to behave as the link text. In this case, it would be the `{meetup.title}`.

`LinkBox` => should have all the layout styles
`LinkOverlay` => is the `<a>` tag that should only wrap the text

Reply:
Ok that's understandable. S0 even though if I wrap the text with `LinkOverlay` the `LinkBox`  will make the other Box  clickable correct? In a sense its kind of like wrapping all the components in a `Form`?

Reply:
Yes exactly, it is useful when you need to create this clickable components (like Cards) which are more than just a text link, and you still want to keep the html semantics in a good shape.